src/core/sys/posix/sys/socket.d

```diff
@@ -578,39 +578,42 @@ else version (Darwin)
 
     struct msghdr
     {
-        void*     msg_name;
-        socklen_t msg_namelen;
-        iovec*    msg_iov;
-        int       msg_iovlen;
-        void*     msg_control;
-        socklen_t msg_controllen;
-        int       msg_flags;
+        void*    msg_name;
+        uint     msg_namelen;
+        iovec*   msg_iov;
+        uint     msg_iovlen;
```

Comment:
Changed from int -> uint according to :
https://opensource.apple.com/source/xnu/xnu-201/bsd/sys/socket.h

Reply:
What version did this change in or has this always been wrong?